### PR TITLE
ODPM-72 - Improve dev docs around migrations and data import

### PR DESCRIPTION
**Pull request by trawick**

### CODE REVIEW

File: docs/data-import.rst [hunk 1]
```diff
@@ -19,17 +21,25 @@ To load the database dump, run:
...
 
 Raw NC Data (slower)
 ____________________
 
-Make sure our NC database is in the right state before importing:
+Command-line
+++++++++++++
+
+Make sure our NC database is in the right state before importing.
+This is not required each time data is imported.
```
Comment (rlconley) [inline docs/data-import.rst]: If it isn't required, how do we determine when to do it?

File: docs/data-import.rst [hunk 2]
```diff
@@ -43,49 +53,78 @@ Run the import command:
...
+Reusing an existing ``--dest`` directory will speed up import.  However,
+if import code has changed since the last time the directory was used, don't
+reuse an existing directory.
 
-.. code-block:: bash
+Admin
++++++
 
-    sudo -u postgres psql -c 'ALTER USER traffic_stops_staging WITH NOSUPERUSER;'
+Follow the "Admin" instructions above under "Local/Development Environment".
```
Comment (rlconley) [inline docs/data-import.rst]: Maybe worth adding a hint about what to enter as source and destination in the admin.

File: docs/data-import.rst [hunk 3]
```diff
@@ -19,17 +21,25 @@ To load the database dump, run:
     unzip traffic_stops_nc_production_20150801.tar.zip
     pg_restore -Ox -d traffic_stops_nc traffic_stops_nc_production.tar
 
-To create the database dump, run:
+Browse https://s3-us-west-2.amazonaws.com/openpolicingdata/ to see what dumps
```
Comment (nmashton) [inline docs/data-import.rst]: I wasn't aware you could do this; this addition is very helpful.

**Comment by nmashton** [comment]
@rlconley's question up there is on point; otherwise, :shipit:, this document satisfies me w.r.t. the questions I have repeatedly asked you.

File: docs/data-import.rst (hunk 3 above)
Comment (trawick) [inline docs/data-import.rst]: FWIW @copelco hasn't stored any db dumps on S3 for a while AFAICT; perhaps that should be one of the end-of-project/go-live tasks

File: docs/data-import.rst (hunk 1 above)
Comment (trawick) [inline docs/data-import.rst]: See updates.  Basically, there's nothing magic about when to create the database and apply migrations unless you've been playing around with it.

File: docs/data-import.rst (hunk 2 above)
Comment (trawick) [inline docs/data-import.rst]: I made the admin instructions in the earlier section more explicit.

**Comment by rlconley** [comment]
:shipit:

**Pull request merged**